nservicebus/azure-service-bus/transaction-support.md

```diff
@@ -7,37 +7,66 @@ tags:
+Note: `SendAtomicWithReceive` level is supported only as from version 7 and when destination and receive queues are in the same namespace.
 
-NServiceBus Azure Service Bus transport configuration is hard-coded to `Serializable` isolation level. Users can't override it.
+The `SendAtomicWithReceive` guarantee is achieved by using `ViaEntityPath` property on outbound message senders. It's value is set to the receiving queue.
 
+If the `ViaEntityPath` is not empty, then messages will be added to the receive queue. The messages will be forwarded to their actual destination (inside the broker) only when the complete operation is called on the received brokered message. The message won't be forwarded if the lock duration limit is exceeded (30 seconds by default) or if the message is explicitly abandoned.
 
-### Versions 6 and above
 
-Azure Service Bus Transport supports `SendAtomicWithReceive`, `ReceiveOnly` and `None` levels.
+#### Transport transaction - Receive Only
 
+The `ReceiveOnly` guarantee is based on the Azure Service Bus Peek-Lock mechanism.
 
-#### Transport transaction - Sends atomic with Receive
+The message is not removed from the queue directly after receive, but it's hidden for, by default , 30 seconds. That prevents other instances from picking it up. If the receiver fails to process the message withing that timeframe or explicitly abandons the message, then the message will become visible again. Other instances will be able to pick it up (effectively works like a rollback).
 
-Note: `SendAtomicWithReceive` level is supported only when destination and receive queues are in the same namespace.
 
-The `SendAtomicWithReceive` guarantee is achieved by using `ViaEntityPath` property on outbound messages. It's value is set to the receiving queue.
+#### Unreliable (Transactions Disabled)
 
-If the `ViaEntityPath` is not empty, then messages will be added to the receive queue. The messages will be forwarded to their actual destination (inside the broker) only when the complete operation is called on the received brokered message. The message won't be forwarded if the lock duration limit is exceeded (30 seconds by default) or if the message is explicitly abandoned.
+When transactions are disabled then NServiceBus the transport uses the [ASB's ReceiveAndDelete mode](https://msdn.microsoft.com/en-us/library/microsoft.servicebus.messaging.receivemode.aspx).
 
+The message is deleted from the queue directly after receive operation completes, before it is processed.
 
-#### Transport transaction - Receive Only
+This mode is highly unrecommended!
```

Comment:
@SimonCropp yes, http://www.dictionary.com/browse/unrecommended
@danielmarbach I'll add a remark that transient exceptions in the cloud are the norm, so running without retry is not a good idea